pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -361,6 +361,9 @@ func isHealthyPod(pod *corev1.Pod) bool {
 		if status.State.Terminated != nil && status.State.Terminated.ExitCode != 0 {
 			return false
 		}
+		if status.RestartCount > 0 {
```

Comment:
Can we make this restart could configurable, or time-bound? 
1-2 restarts may not be bad (10-100 are). But it also may be dependant on time, was this 10 min ago or 4 days ago.